doc/bitcoin-conf.md

```diff
@@ -0,0 +1,11 @@
+# `bitcoin.conf` - Bitcoin configuration file
+
+All command-line options (except for `-conf`) may be specified in a configuration file, and all configuration file options (except for `-includeconf`) may also be specified on the command line. Command-line options override values set in the configuration file and configuration file options override values set in the GUI.
```

Comment:
Perhaps mention that certain values need specifying like `testnet=1` in the configuration, but is just `-testnet` on command-line?